### PR TITLE
Update AdminController.cs

### PR DESCRIPTION
**Pull request by Garpo001**
Fix action on email test  post

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs [hunk 1]
```diff
@@ -49,7 +49,7 @@ public async Task<IActionResult> Index()
             return View();
         }
 
-        [HttpPost]
+        [HttpPost, ActionName(nameof(Index))]
```
Comment (jptissot) [inline src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs]: Won't this break https://github.com/Garpo001/OrchardCore/blob/e353bb0da2c5b5dbe2a217b3ea6d9813a80ac9b5/src/OrchardCore.Modules/OrchardCore.Email/Views/Admin/Index.cshtml#L5 ? @agriffard

Comment (agriffard) [reply src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs]: OK, Action also changed in the form.

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs]: How can this work if the action is "Index" and the form action is "IndexPost"?

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Email/Controllers/AdminController.cs]: Ok you guys renamed it back to index again. But this time fixed the action on the controller. Kind of the same but I don't understand why it kept changing to "index" for the form action.

**Pull request merged**